src/analyses/variable-sensitivity/variable_sensitivity_configuration.cpp

```diff
@@ -41,9 +41,7 @@ vsd_configt vsd_configt::from_options(const optionst &options)
 
   // This should always be on (for efficeny with 3-way merge)
   // Does not work with value set
```

Comment:
If it works, remove the comment!

Reply:
Turns out there's another chunk that should also go 
```
  if(
    options.get_bool_option("value-set") &&
    options.get_bool_option("data-dependencies"))
  {
    throw invalid_command_line_argument_exceptiont{
      "Value set is not currently supported with data dependency analysis",
      "--value-set --data-dependencies",
      "--data-dependencies"};
  }
```